### PR TITLE
Update helm chart to be used with config-controller

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/16590

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by MarcelMue** [comment]
Change looks sensible to me - are you aware that master is only used on AWS?
KVM and Azure are using legacy:
https://github.com/giantswarm/cluster-operator/tree/legacy

**Review by stone-z — APPROVED** [review]
Fine with me. Not sure what the timeline is for draughtsman deprecation but you might need to do the same for legacy

**Comment by corest** [comment]
Thx folks, will test this and then port to legacy

**Comment by corest** [comment]
Works fine. Will merge after merging config repo is merged

File: helm/cluster-operator/values.yaml [hunk 1]
```diff
@@ -13,3 +10,53 @@ pod:
...
+          chart:     "chart-operator"
+          namespace: "giantswarm"
+        # Upgrade force is disabled to avoid affecting customer workloads.
+        coredns:
+          useUpgradeForce: false
+        net-exporter:
+          chart: "net-exporter"
+        # Upgrade force is disabled to avoid affecting customer workloads.
+        nginx-ingress-controller:
+          useUpgradeForce: false
```
Comment (kopiczko) [inline helm/cluster-operator/values.yaml:58]: Was that configured in installations before?

**Review by kopiczko — APPROVED** [review]
This configuration in strings looks strange but if it was configured like that then LGTM

File: helm/cluster-operator/values.yaml (hunk 1 above)
Comment (corest) [reply helm/cluster-operator/values.yaml:58]: yes, I took it from installations

**Review by kubasobon — APPROVED** [review]
LGTM

**Pull request merged**